test_regress/t/t_module_class_static_method.out

```diff
@@ -0,0 +1,5 @@
+%Error: t/t_module_class_static_method.v:18:12: Can't find definition of task/function: 'static_task'
+                                              : ... In instance t
```

Comment:
I believe the error message this used to give was "Unsupported: 'static' class method"

We need something similar to indicate this is unsupported syntax versus something the user botched.

Or, of course, support it which would be nice.

Reply:
Managed to fix it in the latest commit.